sys/include/net/gnrc/sixlowpan/frag/sfr_types.h

```diff
@@ -43,6 +45,9 @@ typedef union {
  *          fragment recovery
  */
 typedef struct gnrc_sixlowpan_frag_sfr_fb {
+#if IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_SFR_CONGURE) || DOXYGEN
+    congure_snd_t *congure;     /**< state object for [CongURE](@ref sys_congure) */
```

Comment:
What does it mean to use SFR without CongURE? 

Or does SFR also work without any congestion control at all?